src/browser/api/window.js

```diff
@@ -506,6 +506,9 @@ Window.create = function(id, opts) {
             let closeEventString = route.window('close-requested', uuid, name);
             let listenerCount = ofEvents.listenerCount(closeEventString);
 
+            // Ensure that any queued app logs are sent to the RVM.
+            flushConsoleMessageQueue();
```

Comment:
Is this supposed to be Window specific or Application specific ? 

On a Multi Window application this operation will be called multiple times.

Reply:
Yeah, this makes more sense as an application-level call. I moved this to just before an application `closed` event is sent to the RVM.